zzzz_modular_occulus/code/game/machinery/dorms_elevator.dm

```diff
@@ -0,0 +1,131 @@
+///// Dorms Elevator/////
+
+//// This code is for the elevator to remove the player from the round and delete all items outside of the preserve items list.
+
+/obj/machinery/button/remote/elevator_panel/proc/despawn_passenger(var/mob/living/user)
+	var/mob/living/passenger = user
+
+	for(var/obj/item/W in passenger)
+		if(W in preserve_items)
+			passenger.drop_from_inventory(W)
```

Comment:
Is there a way to link this to a cryo console, so that special items aren't just there for the taking, or does this do that automagically?

Reply:
The issue with items is that the cryostorage units have their own little pockets for storing special items. I can't really do that for an elevator, since rather than being a single tile pod, they're an area on the map. I'm not really sure what to do about special items dropping. It's better for them to drop on the floor than just get deleted, cus someone could delete the captain's spare which is... bad. Speaking of...

Another bug I forgot to mention was that the list of preserve items was just getting ignored completely. The only thing that never gets deleted is the ID card inside the player's PDA. Everything else is deleted successfully, regardless of whether it's in the preserve list or not. So I need to do some testing to see if I can fix that issue as well.